Source/Core/Common/Assembler/GekkoLexer.h

```diff
@@ -19,6 +19,18 @@ namespace Common::GekkoAssembler::detail
 {
 void ConvertStringLiteral(std::string_view literal, std::vector<u8>* out_vec);
 
+constexpr auto eq_nocase = [](std::string_view str, std::string_view lwr) {
```

Comment:
Can be replaced with `CaseInsensitiveEquals` (from `StringUtil.h`).

Reply:
good idea